dsa-solutions/lc-solutions/0100-0199/0125-valid-palindrome.md

```diff
@@ -27,17 +29,15 @@ Explanation: "amanaplanacanalpanama" is a palindrome.
 ```
 
 **Example 2:**
-
 ```
 Input: "race a car"
 Output: false
 Explanation: "raceacar" is not a palindrome.
 ```
 
 ### Constraints
-
-- The input string may consist of uppercase and lowercase letters, digits, and punctuation symbols.
-- The empty string is considered a valid palindrome.
+- 1 <= s.length <= 2 * 105
```

Comment:
replace `- 1 <= s.length <= 2 * 105` to `- $1 \leq s.length \leq 2 \times 105$`

Reply:
Done